pandas/_libs/src/ujson/python/objToJSON.c

```diff
@@ -106,6 +106,8 @@ typedef struct __TypeContext {
     double doubleValue;
     JSINT64 longValue;
 
+    char bigNum;
```

Comment:
You would want a pointer to a char here, but it's probably easiest to not add another struct member and just write to `cStr` for now

Reply:
Ok! Got rid of this